stablehlo/reference/Element.h

```diff
@@ -41,8 +44,60 @@ class Element {
       : type_(type), value_(std::make_pair(value.real(), value.imag())) {}
 
   Element(const Element &other) = default;
+  Element() = default;
```

Comment:
Why do we need to add a no-parameter constructor?

Reply:
This was is in DotGeneralOp to declare the Element object, but now with this PR, it is no longer needed. Will remove.